.github/workflows/test.yaml

```
@@ -32,22 +32,28 @@ jobs:
           - { py: 2.7, toxenv: py27-epolls, ignore-error: false }
           - { py: 2.7, toxenv: py27-poll, ignore-error: false }
           - { py: 2.7, toxenv: py27-selects, ignore-error: false }
+          - { py: 2.7, toxenv: py27-dnspython1, ignore-error: false }
           - { py: 3.5, toxenv: py35-epolls, ignore-error: false }
           - { py: 3.5, toxenv: py35-poll, ignore-error: false }
           - { py: 3.5, toxenv: py35-selects, ignore-error: false }
+          - { py: 3.5, toxenv: py35-dnspython1, ignore-error: false }
           - { py: 3.6, toxenv: py36-epolls, ignore-error: false }
           - { py: 3.6, toxenv: py36-poll, ignore-error: false }
           - { py: 3.6, toxenv: py36-selects, ignore-error: false }
+          - { py: 3.6, toxenv: py36-dnspython1, ignore-error: false }
           - { py: 3.7, toxenv: py37-epolls, ignore-error: false }
           - { py: 3.7, toxenv: py37-poll, ignore-error: false }
           - { py: 3.7, toxenv: py37-selects, ignore-error: false }
+          - { py: 3.7, toxenv: py37-dnspython1, ignore-error: false }
           - { py: 3.8, toxenv: py38-epolls, ignore-error: false }
           - { py: 3.8, toxenv: py38-openssl, ignore-error: false }
           - { py: 3.8, toxenv: py38-poll, ignore-error: false }
           - { py: 3.8, toxenv: py38-selects, ignore-error: false }
+          - { py: 3.8, toxenv: py38-dnspython1, ignore-error: false }
           - { py: 3.9, toxenv: py39-epolls, ignore-error: false }
           - { py: 3.9, toxenv: py39-poll, ignore-error: false }
           - { py: 3.9, toxenv: py39-selects, ignore-error: false }
+          - { py: 3.9, toxenv: py39-dnspython1, ignore-error: false }
```

```suggestion
          - { py: 3.x, toxenv: dnspython1, ignore-error: false }
```
+ remove combination explosion above

IMHO, it's sufficient to check against dnspython1 on any single Python version.

---

Should I literally write `3.x` here? (Sorry I'm not so familiar)

Otherwise I have updated to only test on Python 2.7 & 3.9.

---

Yeah 27 is fine even if redundant. Also you can press "commit suggestion" button if you agree with proposed code.

---

3.x selects "latest of 3.any"

---

I see. I thought 3.x was invalid so I didn't click the button. Sorry :P